src/contractRpc.ts

```diff
@@ -71,7 +71,11 @@ function extractContractExecutionOutput(
   returnType: TypeDef | null | undefined
 ): QueryContractOutput {
   const data = result.data.toU8a(true);
-  if (!result.flags.isRevert) {
+
+  const contractReturnFlags = (result.flags as any).bits?.toNumber() ?? 0;
```

Comment:
Let's again add a comment here as it's not really obvious why this is required.
@TorstenStueber do you know roughly since when this is required? How come this suddenly changed? If I correctly understand the description in #20 it must a be bug in one of the polkadot.js dependencies that messed up the type/decoding of this parameter (?)

Reply:
I checked now when and how the type changed. It happened directly between the (currently latest) version 13.2.1 and the previous version 13.1.1 ([there are none in between](https://www.npmjs.com/package/@polkadot/api-contract?activeTab=versions)).

When executing

```
console.log(result.flags.toHuman(), result.flags.bits?.toNumber(), result.flags.isRevert)
```

Then on version 13.1.1 I get the following output:
* if not reverted: `[] undefined false`
* if reverted: `[ 'Revert' ] undefined true`

On version 13.2.1 I get the output:
* if not reverted: ` { bits: '0' } 0 undefined `
* if reverted: `{ bits: '1' } 1 undefined`

Reply:
I submitted a bug ticket.

Reply:
Thanks for looking into it and adding the comment 👍